chronoline/chronoline.js

```diff
@@ -472,6 +473,7 @@ function Chronoline(domElement, events, options) {
                 addElemClass(t.paperType, elem.node, 'chronoline-event');
 
                 elem.attr('title', myEvent.title);
+                var $node = jQuery(elem.node);
```

Comment:
would it be possible to wrap this use of `jQuery` in a check? chronoline is supposed to work without jQuery if you don't want fancy tooltips, and it would be nice to maintain that soft dependency.